purity_fb/purity_fb_1dot5/models/policy_patch.py

```diff
@@ -36,88 +36,32 @@ class PolicyPatch(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'name': 'str',
```

Comment:
We should add to the release notes that we removed the unsupported `name` and `id` properties from PolicyPatch.

Reply:
added to docs/index.md, README.md and purity_fb/README.md